grafana/files/grafana.ini

```diff
@@ -20,7 +20,7 @@ logs = {{ server.path.logs }}
 #################################### Server ####################################
 [server]
 # Protocol (http or https)
-;protocol = http
+protocol = {{ server.protocol }}
```

Comment:
If you can change the protocol to `https`, you'll probably also want to be able to set the `cert_file` and `cert_key`.

Reply:
Although it seems like some of your changes have been implemented as well in #36